yin_yang/plugins/icons.py

```diff
@@ -64,3 +67,35 @@ def available_themes(self) -> dict:
                 themes.extend(d.name for d in entries if d.is_dir() and path.isfile(d.path + '/index.theme'))
 
         return {t: t for t in themes}
+
+class _Kde(PluginCommandline):
+    def __init__(self):
+        super().__init__(["/usr/lib/plasma-changeicons", r"{theme}"])
+        self.theme_light = "breeze"
+        self.theme_dark = "breeze-dark"
+
+    @property
+    def available_themes(self) -> dict:
+        themes = []
+
+        for icon_theme in theme_directories:
+            if not path.isdir(icon_theme):
+                continue
+
+            for icon_theme_folder in scandir(icon_theme):
+                if not all(
+                    (
+                        icon_theme_folder.is_dir(),
+                        path.isfile(icon_theme_folder.path + "/index.theme"),
+                        path.isfile(icon_theme_folder.path + "/icon-theme.cache"),
+                    )
+                ):
+                    continue
+
+                theme_parser = configparser.ConfigParser(strict=False)
+                theme_parser.read(icon_theme_folder.path + "/index.theme")
```

Comment:
Similar to the above: `read(icon_theme_folder / 'index.theme')`

Reply:
See above